package.json

```diff
@@ -9,9 +9,9 @@
   },
   "engines": {
     "node": "^20.14.0",
-    "pnpm": "^9.7.0"
+    "pnpm": "^10.11.1"
```

Comment:
You'll also need to update the pnpm version in the MODULE.bazel file otherwise things might be wonky when it's run from clean in bazel